utility.py

```diff
@@ -264,3 +267,46 @@ def tag_user(user, tag, channel):
                 queue_message_to_one(messages.tag_message(get_display_name(user_id), tag), channel)
             else:
                 queue_message_to_one(messages.already_tag_message(get_display_name(user_id), tag), channel)
+
+### Banphrase API ###
+
+def check_banphrase(message, channel_name):
+    headers = { 'User-Agent': 'huwobot (https://huwobot.com/)' }
+    params = (('message', message),)
+
+    banphrase_api =  db(opt.CHANNELS).find_one({'name': channel_name})['banphrase_api']
+
+    if len(banphrase_api) == 0:
+        return False
+
+    try:
+        time.sleep(random.uniform(0.1, 1))
+        response = requests.post('https://' + banphrase_api + '/api/v1/banphrases/test', headers=headers, params=params).json()
+        return response
+    except:
+        raise Exception()
```

Comment:
Don't use generic exceptions, especially in a try/except. Remove this try/except unless you need to add clean up after exceptions happen like
```python
try: 
    ...
except: 
    # clean up here
    raise  # didn't handle the exception, raise it again
```